src/server/graphql/mutations/login.js

```diff
@@ -43,6 +43,17 @@ const login = {
       const user = await dataLoader.get('users').load(viewerId);
       // LOGIN
       if (user) {
+        /*
+         * The segment docs are inconsistent, and warn against sending
+         * identify() on each log in. However, calling identify is the
+         * only way to synchronize changing user properties with certain
+         * services (such as Hubspot). After checking with support
+         * and combing the forums, it turns out sending identify()
+         * on each login is just fine.
+         *
+         * See also: https://community.segment.com/t/631m9s/identify-per-signup-or-signin
+         */
+        await sendSegmentIdentify(user.id);
```

Comment:
we should probably identify new users, too? this will only identify you after you log out for your first time. i don't think that's the intent?

Reply:
Good catch here